### PR TITLE
Milestone 2: My portfolio projects

### PR DESCRIPTION
**Pull request by aliabbani**
## Portfolio Project: milestone 2. Portfolio mobile version.
In this milestone we:

- added a section for the project portfolio. 
- added a section for personal info and languages.
- added HTML & CSS files.
- Used mobile-first approach.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -77,4 +82,248 @@
...
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+
+  /* Inside Auto Layout
+flex: none;
+order: 0;
+flex-grow: 0;
+margin: 0px 0px;  */
+}
```
Comment (arslanbisharat) [inline style.css]: - [ ]Please remove commented code to make your code clean.

File: style.css [hunk 2]
```diff
@@ -77,4 +82,248 @@
...
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563
+
+  /* Inside Auto Layout
+flex: none;
+order: 0;
+flex-grow: 0;
+margin: 0px 0px;  */
+}
```
Comment (arslanbisharat) [inline style.css]: - [ ]Please remove commented code to make your code clean.

File: style.css [hunk 3]
```diff
@@ -77,4 +82,248 @@
...
+  height: 24px;
+  margin-right: 5%;
+
+  /* Inside Auto Layout
+flex: none;
+order: 0;
+flex-grow: 0;
+margin: 0px 8px; */
+}
+
```
Comment (arslanbisharat) [inline style.css]: - [ ]Please remove commented code to make your code clean.

File: style.css [hunk 4]
```diff
@@ -77,4 +82,248 @@
...
+  margin-top: 5%;
+  margin-bottom: 5%;
+  margin-left: 8%;
+
+  /* Inside Auto Layout
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0px 10px; */
+}
```
Comment (arslanbisharat) [inline style.css]: - [ ]Please remove commented code to make your code clean.

File: style.css [hunk 5]
```diff
@@ -77,4 +82,248 @@
...
+  line-height: 160%;
+  color: #344563;
+  margin-left: 8%;
+
+  /* Inside Auto Layout
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0px;  */
+}
```
Comment (arslanbisharat) [inline style.css]: - [ ]Please remove commented code to make your code clean.

File: style.css [hunk 6]
```diff
@@ -77,4 +82,248 @@
...
+  color: #172b4d;
+  margin-left: 8%;
+
+  /* Inside Auto Layout
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 0px;  */
+}
+
```
Comment (arslanbisharat) [inline style.css]: - [ ]Please remove commented code to make your code clean.

File: index.html [hunk 7]
```diff
@@ -33,6 +33,127 @@ <h2 class="name">I’m Ali<br>Glad to see you!</h2>
...
+          </div>
+          <p class="middle-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <img src="images/Snapshoot-Portfolio.png" alt="Snapshoot Portfolio" class="middle-images">
+          <div class="middle-button">
+            <button class="html-button">html</button>
+            <button class="css-button">css</button>
+            <button class="js-button">javaScript</button>
+          </div>
+          <button type="button" class="button-see-project">See Project</button>
+        </article>
```
Comment (arslanbisharat) [inline index.html]: - [ ]The list of projects should use a` ul tag` to group them, and `li` per project

File: index.html [hunk 8]
```diff
@@ -33,6 +33,127 @@ <h2 class="name">I’m Ali<br>Glad to see you!</h2>
...
+          </div>
+          <p class="middle-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <img src="images/Snapshoot-Portfolio2.png" alt="Snapshoot Portfolio 2" class="middle-images">
+          <div class="middle-button">
+            <button class="html-button">html</button>
+            <button class="css-button">css</button>
+            <button class="js-button">javaScript</button>
+          </div>
+          <button type="button" class="button-see-project">See Project</button>
+        </article>
```
Comment (arslanbisharat) [inline index.html]: - [ ]The list of projects should use a` ul tag` to group them, and `li` per project

File: index.html [hunk 9]
```diff
@@ -33,6 +33,127 @@ <h2 class="name">I’m Ali<br>Glad to see you!</h2>
...
+          </div>
+          <p class="middle-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <img src="images/Snapshoot-Portfolio3.png" alt="Snapshoot Portfolio 3" class="middle-images">
+          <div class="middle-button">
+            <button class="html-button">html</button>
+            <button class="css-button">css</button>
+            <button class="js-button">javaScript</button>
+          </div>
+          <button type="button" class="button-see-project">See Project</button>
+        </article>
```
Comment (arslanbisharat) [inline index.html]: - [ ]The list of projects should use a` ul tag` to group them, and `li` per project

File: index.html [hunk 10]
```diff
@@ -33,6 +33,127 @@ <h2 class="name">I’m Ali<br>Glad to see you!</h2>
...
+          </div>
+          <p class="middle-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <img src="images/Snapshoot-Portfolio4.png" alt="Snapshoot Portfolio 4" class="middle-images">
+          <div class="middle-button">
+            <button class="html-button">html</button>
+            <button class="css-button">css</button>
+            <button class="js-button">javaScript</button>
+          </div>
+          <button type="button" class="button-see-project">See Project</button>
+        </article>
```
Comment (arslanbisharat) [inline index.html]: - [ ]The list of projects should use a` ul tag` to group them, and `li` per project

File: index.html [hunk 11]
```diff
@@ -33,6 +33,127 @@ <h2 class="name">I’m Ali<br>Glad to see you!</h2>
...
+            <img src="images/Counter.png" alt="">
+            <span class="canopy-2">2015</span>   
+          </div>
+          <p class="middle-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <img src="images/Snapshoot-Portfolio4.png" alt="Snapshoot Portfolio 4" class="middle-images">
+          <div class="middle-button">
+            <button class="html-button">html</button>
+            <button class="css-button">css</button>
+            <button class="js-button">javaScript</button>
+          </div>
```
Comment (arslanbisharat) [inline index.html]: - [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`

File: index.html [hunk 12]
```diff
@@ -33,6 +33,127 @@ <h2 class="name">I’m Ali<br>Glad to see you!</h2>
...
+            <img src="images/Counter.png" alt="">
+            <span class="canopy-2">2015</span>   
+          </div>
+          <p class="middle-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <img src="images/Snapshoot-Portfolio3.png" alt="Snapshoot Portfolio 3" class="middle-images">
+          <div class="middle-button">
+            <button class="html-button">html</button>
+            <button class="css-button">css</button>
+            <button class="js-button">javaScript</button>
+          </div>
```
Comment (arslanbisharat) [inline index.html]: - [x] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`

File: index.html [hunk 13]
```diff
@@ -33,6 +33,127 @@ <h2 class="name">I’m Ali<br>Glad to see you!</h2>
...
+            <img src="images/Counter.png" alt="">
+            <span class="canopy-2">2015</span>   
+          </div>
+          <p class="middle-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <img src="images/Snapshoot-Portfolio2.png" alt="Snapshoot Portfolio 2" class="middle-images">
+          <div class="middle-button">
+            <button class="html-button">html</button>
+            <button class="css-button">css</button>
+            <button class="js-button">javaScript</button>
+          </div>
```
Comment (arslanbisharat) [inline index.html]: - [x] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`

File: index.html [hunk 14]
```diff
@@ -33,6 +33,127 @@ <h2 class="name">I’m Ali<br>Glad to see you!</h2>
...
+            <img src="images/Counter.png" alt="">
+            <span class="canopy-2">2015</span>   
+          </div>
+          <p class="middle-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <img src="images/Snapshoot-Portfolio.png" alt="Snapshoot Portfolio" class="middle-images">
+          <div class="middle-button">
+            <button class="html-button">html</button>
+            <button class="css-button">css</button>
+            <button class="js-button">javaScript</button>
+          </div>
```
Comment (arslanbisharat) [inline index.html]: - [x] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi @aliabbani, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] You have an empty file in your project directory. Please remove that to make your project clean. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: style.css (hunk 1 above)
Comment (aliabbani) [reply style.css]: ok

File: style.css (hunk 6 above)
Comment (aliabbani) [reply style.css]: ok

File: index.html (hunk 14 above)
Comment (aliabbani) [reply index.html]: ok

File: index.html [hunk 15]
```diff
@@ -33,6 +33,127 @@ <h2 class="name">I’m Ali<br>Glad to see you!</h2>
...
+          <p class="middle-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <img src="images/Snapshoot-Portfolio.png" alt="Snapshoot Portfolio" class="middle-images">
+          <ul class="middle-button">
+            <li class="html-button">html</li>
+            <li class="css-button">css</li>
+            <li class="js-button">javaScript</li>
+          </ul>
+          <button type="button" class="button-see-project">See Project</button>
+        </article>
```
Comment (Lameck1) [inline index.html]: - [x] The previous code review pointed out these sections specifically and suggested that all projects be in a `<ul>` tag and each project be in an `<li>` tag. As this was not optional, let's kindly refactor it. Remember this is for all projects listed.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi @aliabbani, 

You've done well making some changes requested by the previous reviewer 👏.
It's like we missed some of the requested changes though. Let's take a second look, please.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 15 above)
Comment (aliabbani) [reply index.html]: Thank you for your explanation, I will solve the issue.

File: style.css (hunk 4 above)
Comment (rloterh) [reply style.css]: Well resolved

File: style.css (hunk 3 above)
Comment (rloterh) [reply style.css]: Well resolved

File: style.css (hunk 2 above)
Comment (rloterh) [reply style.css]: Well resolved

File: style.css (hunk 1 above)
Comment (rloterh) [reply style.css]: Well resolved

File: index.html (hunk 15 above)
Comment (rloterh) [reply index.html]: Well resolved

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Abbani, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewers. There is nothing else to say other than... it's time to merge it 👍.
Congratulations! 🥂

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
__As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have no more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).__

**Pull request merged**